.env.example

```diff
@@ -1,5 +1,9 @@
- # The environment to use `development`, `testing`, `staging`, `production`
- STELLAR_SCAFFOLD_ENV=development
+# The environment to use `development`, `testing`, `staging`, `production`
+STELLAR_SCAFFOLD_ENV=development
+
+# Location of the config files for this project for the scaffold stellar CLI.
```

Comment:
This fixes (for new users/people starting from init) the issues of floating "identity" and "alias" folders, and for scaffold stellar to know what config you're using.